code/modules/surgery/organs/liver.dm

```diff
@@ -217,6 +220,7 @@
 	name = "reagent processing crystal"
 	icon_state = "liver-p"
 	desc = "A large crystal that is somehow capable of metabolizing chemicals, these are found in plasmamen."
+	liver_biotype = MOB_MINERAL
```

Comment:
Every single ``affected_biotype`` currently is ``MOB_ORGANIC``, does this do anything?

Reply:
Yes. This important because it is what stops plasmamen from taking liver damage from plasma and other organic toxins.

`if(!(liver_biotype & toxin.affected_biotype)) //this particular toxin does not affect this biotype
	continue `

For plasmamen who do not have the MOB_ORGANIC liver biotype this will cause it to skip over all toxins that do not have the MOB_MINERAL affected_biotype flag.

Reply:
As for why not use mob biotype here, it's because transplanted and cybernetic livers are a thing. So the liver biotype should be considered independently. All livers are organic except the plasmaman one currently.

More types of livers may be added this way if you want it to be able to process a specific affected_biotype.